--- /home/tbonetti/.oh-my-zsh/themes/agnoster.zsh-theme	2018-11-06 11:07:55.750543610 +0100
+++ ../zsh-custom/agnoster-custom.zsh-theme	2018-11-06 21:20:31.239698290 +0100
@@ -224,6 +224,10 @@
   [[ -n "$symbols" ]] && prompt_segment black default "$symbols"
 }
 
+clock() {
+  echo -n " ⌚ %{%F{red}%}%*%{%f%}"
+}
+
 ## Main prompt
 build_prompt() {
   RETVAL=$?
@@ -235,6 +239,8 @@
   prompt_bzr
   prompt_hg
   prompt_end
+  clock
 }
 
-PROMPT='%{%f%b%k%}$(build_prompt) '
+PROMPT='%{%f%b%k%}$(build_prompt)
+$ '
